src/Compilers/CSharp/Portable/Binder/DecisionDagBuilder.cs

```diff
@@ -448,7 +448,7 @@ private static void Assert(bool condition, string message = null)
                 else
                 {
                     // This occurs in error cases.
-                    Debug.Assert(recursive.HasAnyErrors);
+                    //Debug.Assert(recursive.HasAnyErrors);
```

Comment:
I won't merge this change. It's to unblock manual validation in IDE.
Filed https://github.com/dotnet/roslyn/issues/26467

Reply:
Fix for that is in https://github.com/dotnet/roslyn/issues/26481